ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Problem:
```

Comment:
My concern is the more we condense the more reporters will leave stuff out and we'll have to ask for it again.